mcp/mcp.c

```diff
@@ -296,26 +251,19 @@ static void mcp_5hz_routines(void)
 {
     watchdog_ping();
     // Tickles software WD 2.5x as fast as timeout
-
-    update_sun_sensors();
```

Comment:
@ianlowe13 no sun sensors anymore?

Reply:
Probably meant to comment it out. Will fix in iandev.

Reply:
Looking more into it - commenting out is technically correct (or removing temporarily). As it stands, the sunsensor module is dependent on the BLAST-TNG labjack architecture which may vary significantly from the TIM build. We will likely be able to use these functions in a similar manner but they will need to be updated when TIM constructs the sun sensor readout as these functions pertain to specifically processing the labjack data and saving it.

Reply:
Currently the LabJack child files all live in legacy mcp, while the parent files that contain all of the necessary structure for setting them up within the code remains. Simple translations and new file builds will be the desired outcome rather than full reuse as the relay control hardware will be different from TNG for example. Sunsensors may be a 1:1, however more information from Penn on the hardware side is necessary.